src/components/Clock/Clock.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import './Clock.scss';
+import PropTypes from 'prop-types';
+
+export class Clock extends React.Component {
+  state = {
+    date: '',
+    clockName: this.props.name,
+    intervalId: undefined,
```

Comment:
```suggestion
    intervalId: null,
```
Значение undefined обычно не присваивается, если мы хотим обозначить временное отсутствие значения в переменной - пишем null. Ещё, в принципе, может и не нужно добавлять intervalId в state, ты же его в рендере не используешь. Можно просто записать вне, как переменную

Reply:
Illia is right